config/locales/en.yml

```diff
@@ -239,7 +242,10 @@ en:
         custom_select_error: Select yes if your basic care needs are being met at the moment
       dietary_requirements:
         title: Do you have any special dietary requirements?
-        hint: For example, you're a vegetarian or you have a food allergy.
+        hint: |
+          <p class="govuk-hint app-hint">For example, you're a vegetarian or you have a food allergy.</p>
+          <br /><br />
+          <p class="govuk-inset-text">For now, food supplies will be delivered in standard packages. Check the ingredients lists carefully if you have a food allergy or follow a special diet. If you have special dietary requirements, some of the food may not meet them.</p>
```

Comment:
<span class="govuk-inset-text">
    <p class="govuk-body">...</p>
</span>

Reply:
Thanks Nick - is this correct now? https://github.com/alphagov/govuk-coronavirus-vulnerable-people-form/pull/113/commits/388cc68e88c2bc7bc4d0d44180cad19112000ab3